packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx

```diff
@@ -23,17 +23,28 @@ export interface MeetingCallControlBarProps {
   onPeopleButtonClicked: () => void;
   mobileView: boolean;
   disableButtonsForLobbyPage: boolean;
+  meetingCallControlOptions?: MeetingCallControlOptions;
 }
 
 /**
  * @private
  */
 export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.Element => {
-  // Disable the default participants button as meetings composite has its own participants button.
+  // Set the desired control buttons from the meetings composite. particiapantsButton is always false since there is the peopleButton.
   let callControlsOptions: CallControlOptions | false = {
     participantsButton: false,
-    screenShareButton: props.mobileView ? false : { disabled: props.disableButtonsForLobbyPage }
+    screenShareButton:
```

Comment:
This can be simplified as 

const callControlOptions = {
  ...props.meetingCallControlOptions,
  screenShareButton: [CONTINUE TO OVERRIDE WHAT YOU NEED]
}

Reply:
oh fun! yeah I'll update that!